qa/workunits/rbd/cli_generic.sh

```diff
@@ -93,6 +93,8 @@ test_others() {
     rbd info testimg5 | grep 'size 256 MB'
     rbd snap ls testimg5 | grep -v 'SNAPID' | wc -l | grep 1
     rbd snap ls testimg5 | grep '.*snap2.*'
+    rbd snap rm testimg4@snap2
+    rbd snap rm testimg5@snap2
```

Comment:
Just wonder, don't we want to make this conditional, i.e. remove only if deep-flatten is not enabled? Something like below:
```
rbd info testimg4 | grep 'features:.*deep-flatten' || rbd snap rm testimg4@snap2
...
```

Reply:
:+1: